src/Stores/StoreHandler.ts

```diff
@@ -0,0 +1,188 @@
+/* eslint-disable no-param-reassign */
+/* eslint-disable no-console */
```

Comment:
Do we want to disable these rules for the whole file?
If yes please add a comment why.

Reply:
We need no-param-reassign as temporary fix to add the userId to items as long we don't have working user auth